src/tsconfig.ts

```diff
@@ -230,3 +252,16 @@ function isFile (stats: fs.Stats | void) {
 function isDirectory (stats: fs.Stats | void) {
   return stats ? (stats as fs.Stats).isDirectory() : false
 }
+
+/**
+ * Simple object merging
+ */
+function mergeObjects (target: any, source: any): any {
+  for (let key of Object.keys(source)) {
+    if (source[key] instanceof Object) {
+      Object.assign(source[key], mergeObjects(target[key], source[key]))
```

Comment:
This is kind of confusing, you're assigning backward onto the source and then copying back onto the target in a roundabout way - you could just use an `else` statement and make both assign to `target` and delete the final `Object.assign` to avoid additional work.